packages/website/src/content/04-03-touchpoint-Buttons.md

```diff
@@ -11,7 +11,11 @@
 
 ### Text Button Example
 
-<img src="/images/Touchpoint-Text-Button.png" alt="Text Button with Left Arrow Icon" style="max-width: 40%;">
+<div class="launch-touchpoint-button"
```

Comment:
We should make this into a React component-looking thing `<TouchpointTrigger title="" label="" description="" context={{}} />`, this would make rendering logic simpler.